.dockerignore

```diff
@@ -0,0 +1,6 @@
+npm-debug.log
+node_modules
+*.swp
+*.swo
+data
+*.DS_Store
```

Comment:
Correct me if I am wrong, but shouldn't this be consistent with .gitignore file?